packages/odyssey-react-mui/src/Select.tsx

```diff
@@ -251,19 +323,63 @@ const Select = <
             return null;
           }
 
-          return <Chip key={item} label={selectedOption.text} />;
+          return (
+            <MuiChip
+              key={item}
+              label={
+                <>
+                  {selectedOption.text}
+                  {!isPlaceholder &&
+                    controlledStateRef.current === CONTROLLED &&
+                    hasMultipleChoices && (
+                      <PlaceholderIcon
+                        odysseyDesignTokens={odysseyDesignTokens}
+                      />
+                    )}
+                </>
+              }
+              tabIndex={-1}
+              onDelete={
+                isPlaceholder
+                  ? controlledStateRef.current === CONTROLLED
+                    ? () => removeSelection(item)
+                    : undefined
+                  : undefined
+              }
+              deleteIcon={
+                // We need to stop event propagation on mouse down to prevent the deletion
+                // from being blocked by the Select list opening, and also ensure that
+                // the pointerEvent is registered even when the parent's are not
+                <CloseCircleFilledIcon
+                  sx={{ pointerEvents: "auto" }}
+                  onMouseDown={(event) => event.stopPropagation()}
```

Comment:
Why are we stopping propagation here? Not saying we don't nee--

I see. Can you move the comment down over this line, so it's more clear?